### ISSUE TITLE
Add tabindex=-1 for Headings to make them focusable with the skiplink

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#3935]
**Is your feature request related to a problem? Please describe.**
Currently there is a skip link from the navigation through to the h1 on each page type. These h1s should have `tabindex="-1"` to ensure they're focusable.

**Already done**:
- Front pages `https://www.bbc.com/igbo`

**Need to do** (page types and example pages)
- Articles: `https://www.test.bbc.co.uk/news/articles/c0g992jmmkko` 
- Live Radio pages : `https://www.test.bbc.com/afaanoromoo/bbc_afaanoromoo_radio/liveradio`
- Error pages: `https://www.test.bbc.com/afaanoromoo/articles/c0000000000o`

**Describe the solution you'd like**
Add the attribute `tabindex="-1"` to the h1 of the page, where the `id="content"` already should be. 

**Describe alternatives you've considered**
N/A - this is necessary for accessibility of the skip link usage.

**Testing notes**
[Tester to complete]

Dev insight: Ensure snapshot tests are updated and the relevant attribute is tested in cypress - alongside the `id="content"`.
e.g. 
```
cy.get('h1')
              .should('have.lengthOf', 1)
              .should('have.attr', 'id', 'content')
              .should('have.attr', 'tabindex', '-1');
```

Should be tested using a keyboard to navigate through the page and check that when the skip link is selected, the h1 has focus. This also should be tested with all the [screen readers we support](https://bbc.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology), check that when the skip link is activated that the h1 then receives focus and is read out.

- [x] This feature is expected to need manual testing.

**Additional context**
